### PR TITLE
#2342 [seo-module] Create

### PR DESCRIPTION
**Pull request by diashalabi**

### CODE REVIEW

File: addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php [hunk 1]
```diff
@@ -32,6 +32,7 @@
 use Visiosoft\LocationModule\Village\VillageModel;
 use Visiosoft\PackagesModule\Package\PackageModel;
 use Visiosoft\ProfileModule\Adress\Contract\AdressRepositoryInterface;
+use Visiosoft\SeoModule\Legend\Command\AddMetaData;
```
Comment (fatihalp) [inline addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php:35]: if seo is not installed? @diashalabi

**Review by fatihalp — CHANGES_REQUESTED** [review]
if seo is not installed? @diashalabi

File: addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php [hunk 2]
```diff
@@ -517,14 +523,32 @@ public function view($seo, $id = null)
                     $coverPhoto = \Illuminate\Support\Facades\Request::root() . '/' . $adv->cover_photo;
                 }
             }
+
+            $metaTitle = $adv->name . " " . end($categories)['name'] . ' ' . setting_value('streams::domain');
+            $metaDesc = strip_tags($adv->advs_desc, '');
+
+            if (is_module_installed('visiosoft.module.seo')) {
```
Comment (diashalabi) [inline addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php:530]: I am checking if seo is installed here

File: addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php (hunk 1 above)
Comment (diashalabi) [reply addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php:35]: https://github.com/openclassify/openclassify/pull/740#discussion_r511009308

Comment (diashalabi) [reply addons/default/visiosoft/advs-module/src/Http/Controller/advsController.php:35]: I tested if the module is not installed no error no problem

**Pull request merged**